requirements.txt

```diff
@@ -0,0 +1 @@
+python-keycloak==2.5.0
```

Comment:
@dimakis not sure if we want to use requirements.txt but I wanted to setup something. 
If you have other better tools let me know